app/controllers/users_controller.rb

```diff
@@ -8,7 +8,7 @@
 class UsersController < ApplicationController
 
   skip_before_filter :authorize_mini_profiler, only: [:avatar]
-  skip_before_filter :check_xhr, only: [:show, :password_reset, :update, :account_created, :activate_account, :perform_account_activation, :user_preferences_redirect, :avatar, :my_redirect, :toggle_anon, :admin_login, :confirm_admin]
+  skip_before_filter :check_xhr, only: [:show, :password_reset, :email_login, :update, :account_created, :activate_account, :perform_account_activation, :user_preferences_redirect, :avatar, :my_redirect, :toggle_anon, :admin_login, :confirm_admin]
```

Comment:
We should not skip `check_xhr` for `email_login` here as well since I believe we are accessing this method via AJAX request.